action.yml

```diff
@@ -0,0 +1,202 @@
+---
+name: Steampipe Check
+author: Turbot
+description: Run Steampipe mod controls and create annotations for Infrastructure as Code controls.
+branding:
+  color: red
+  icon: shield
+inputs:
+  mod-url:
+    description: "Mod URL to clone. Examples: https://github.com/turbot/steampipe-mod-aws-compliance"
+    required: true
+  mod-branch:
+    description: "Mod branch or tag to clone. Defaults to 'main'. Examples: main, release/v0.7, 0.5, 1.0."
+    required: false
+    default: main
+  checks:
+    description: Space separated list of benchmarks and controls to run. Can also pass in as a multi-line string. Defaults to 'all'.
+    required: false
+    default: all
+  artifact-exports:
+    description: "Comma separated export types to save as artifacts. Example: 'json,csv'. Defaults to 'csv'. Valid values: 'asff,csv,html,json,md,nunit13,sps'."
+    required: false
+    default: csv
+  additional-args:
+    description: Space separated args to add to the 'steampipe check' command.
+    required: false
+  snapshot-visibility:
```

Comment:
If a user wants to specify a snapshot title, do they need to use `additional-args`?